apps/web-giddh/src/app/welcome/welcome.component.html

```diff
@@ -101,10 +101,10 @@ <h2>Welcome To Giddh! </h2>
 
                                 <label for="gstinNum" class="d-block">{{formFields['taxName'].label}}<sup
                                         class="text-danger">*</sup></label>
-                                <input type="text" [ngClass]="{'gstinWidth' : isbranch}" class="form-control"
+                                <input type="text" [ngClass]="{'gstinWidth' : isbranch || isOnBoardingInProgress}" class="form-control"
```

Comment:
Remove Min and Max length attributes since we are using regular expressions to validate.